src/Kmd.Logic.Audit.Client.SerilogAzureEventHubs/SerilogAzureEventHubsAuditClientConfigurationExtensions.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using Serilog;
+
+namespace Kmd.Logic.Audit.Client.SerilogAzureEventHubs
+{
+    public static class SerilogAzureEventHubsAuditClientConfigurationExtensions
+    {
+        public static LoggerConfiguration DefaultConfiguration(this SerilogAzureEventHubsAuditClientConfiguration config)
```

Comment:
I think we should use a verb in this method name.

https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/names-of-type-members#names-of-methods

We are not the framework, but most people follow this idea.

```suggestion
        public static LoggerConfiguration CreateDefaultConfiguration(this SerilogAzureEventHubsAuditClientConfiguration config)
```

Reply:
Agreed. Changed